website/content/docs/concepts/policies.mdx

```diff
@@ -800,32 +800,36 @@ authenticated user.
+| [sys/internal/inspect/router](/vault/api-docs/system/inspect/router  )                                                                                 | GET               | Inspect the internal components of Vault's router                                                                                                                                      |
 | [sys/leases/lookup/:prefix](/vault/api-docs/system/leases#list-leases)                                                                                 | LIST              | List lease IDs                                                                                                                                      |
-| [sys/leases/revoke-force/:prefix](/vault/api-docs/system/leases#revoke-force)                                                                          | PUT               | Revoke all secrets or tokens ignoring backend errors                                                                                                |
-| [sys/leases/revoke-prefix/:prefix](/vault/api-docs/system/leases#revoke-prefix)                                                                        | PUT               | Revoke all secrets generated under a given prefix                                                                                                   |
-| [sys/plugins/catalog/:type/:name](/vault/api-docs/system/plugins-catalog#register-plugin)                                                              | GET, PUT, DELETE  | Register a new plugin, or read/remove an existing plugin                                                                                            |
+| [sys/leases/revoke-force/:prefix](/vault/api-docs/system/leases#revoke-force)                                                                          | POST              | Revoke all secrets or tokens ignoring backend errors                                                                                                |
+| [sys/leases/revoke-prefix/:prefix](/vault/api-docs/system/leases#revoke-prefix)                                                                        | POST              | Revoke all secrets generated under a given prefix                                                                                                   |
+| [sys/plugins/catalog/:type/:name](/vault/api-docs/system/plugins-catalog#register-plugin)                                                              | GET, POST, DELETE | Register a new plugin, or read/remove an existing plugin                                                                                            |
 | [sys/raw](/vault/api-docs/system/raw#list-raw)                                                                                                         | LIST, GET         | Returns a list of keys for a given path prefix                                                                                                      |
```

Comment:
Verbs: Actually GET, LIST, POST, DELETE

Description: e.g. `Provides raw access to Vault's underlying storage layer`

Reply:
I will open a separate ticket to review this as it only shows list and get requiring sudo at https://developer.hashicorp.com/vault/api-docs/system/raw#list-raw